views.py

```diff
@@ -6,22 +6,41 @@
 from django.contrib.admin.views.decorators import staff_member_required
 from django.contrib import messages
 from django.db.models import OuterRef, Subquery
+from django.views.decorators.http import require_POST
 from django.utils.decorators import method_decorator
+from django.template import Template, RequestContext, Context
 
 from plugins.consortial_billing import utils, \
      logic, models as supporter_models, plugin_settings, forms
 from plugins.consortial_billing.notifications import notify
 
 from core.views import GenericFacetedListView
 from core.models import Account
+from journal.models import Journal
 from core.model_utils import search_model_admin
 from cms import models as cms_models
+from core.models import SettingValue
 from utils.logger import get_logger
+from utils.logic import get_current_request
 from security.decorators import base_check_required
+from submission import models as submission_models
+
 
 logger = get_logger(__name__)
 
 
+def hourglass(template):
+    """
+    A helper for accessing templates that only exist in Hourglass,
+    with a fallback template to explain this if needed.
+    """
+    request = get_current_request()
+    if request.press.theme == 'hourglass':
+        return template
+    else:
+        return 'consortial_billing/requires_hourglass.html'
```

Comment:
Can we use a better name for this function? we probably want to reserve the name `hourglass` for an eventual module name and it would be good to be able to understand what the function does without having to read the definition.